configs/bedrock.init

```diff
@@ -36,7 +36,7 @@ if [ -n "$OTHERNODES" ]; then
     FLAGS="-peerList $OTHERNODES"
 fi
 
-DAEMON_OPTS="-fork -nodeName $THISNODE -db $BEDROCK_DB_PATH -serverHost 0.0.0.0:8888 -nodeHost 0.0.0.0:8889 -priority $PRIORITY -pidfile $PIDFILE -quorumCheckpoint 100 -readThreads 4 -plugins status,db,jobs,cache,mysql $FLAGS"
+DAEMON_OPTS="-fork -nodeName $THISNODE -db $BEDROCK_DB_PATH -serverHost 0.0.0.0:8888 -nodeHost 0.0.0.0:8889 -priority $PRIORITY -pidfile $PIDFILE -workerThreads 1 $FLAGS"
```

Comment:
same as above, we should remove the workerThreads flag and let bedrock find the number of cores to use.